src/ceph.in

```diff
@@ -715,8 +715,8 @@ def ping_monitor(cluster_handle, name, timeout):
         return 1
 
     mon_id = name[len('mon.'):]
+    run_in_thread(cluster_handle.connect, timeout=timeout)
```

Comment:
there is no need to call `cluster_handle.connect` before calling `cluster_handle.ping_monitor`. please check the implementation of `MonClient::ping_monitor()`.

Reply:
Yes, I check the implementation of MonClient::ping_monitor() and MonMap::build_initial(). 
Before connected to the cluster, the monmap is like this:
```
0: 10.48.43.147:6789/0 mon.a
1: 10.48.43.178:6789/0 mon.b
2: 10.48.43.184:6789/0 mon.c
```
After connected to the cluster, the monmap is like this:
```
0: 10.48.43.147:6789/0 mon.h7515
1: 10.48.43.178:6789/0 mon.h7540
2: 10.48.43.184:6789/0 mon.h7550
```
So when using the command "ceph ping  mon.h7515" get error output and the command "ceph ping mon.a" get correct output. In MonClinet::ping_monitor(), the monmap is like {<noname-a, ip1>, <noname-b, ip2>, <noname-c, ip3>}, "h7515" is not in the map but "a", “b" or "c" in the monmap.

And the command "ceph ping mon.*" get correct output, because it connect to the cluster and get the correct monmap.

So we don't need to connect to the cluster, but we do need get the new-updated monmap.